secant.xcodeproj/project.pbxproj

```diff
@@ -12,6 +12,8 @@
 		0D1922ED26BDE0C600052649 /* AppRouter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0D1922EC26BDE0C600052649 /* AppRouter.swift */; };
 		0D1922F226BDE29300052649 /* ZcashSDKStubs.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0D1922F126BDE29300052649 /* ZcashSDKStubs.swift */; };
 		0D1922F826BDEB3500052649 /* MockServices.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0D1922F726BDEB3500052649 /* MockServices.swift */; };
+		0D24267726E9727C00DA2A47 /* WelcomeScreenTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0D24267626E9727C00DA2A47 /* WelcomeScreenTests.swift */; };
```

Comment:
Just an FYI it looks like there was some accidentally added code here.

Reply:
ahggg,  you had one job `git`. Thanks for pointing this out